patches/chromium/ssl_security_state_tab_helper.patch

```diff
@@ -20,17 +20,22 @@ index 887cdd11fbbd9df44edef1548a89891c5cf23cee..12da72449622289ec467d284f616bbf9
  #include "chrome/browser/safe_browsing/safe_browsing_service.h"
  #include "chrome/browser/safe_browsing/ui_manager.h"
 +#endif
+ #include "chrome/browser/ssl/tls_deprecation_config.h"
```

Comment:
This file should be included in the sources list under `chromium_src/BUILD.gn`

Reply:
Is it worth adding a deletion to the patch? My intuition is to try and leave as much Chromium source unchanged in the patch as possible (but I'm new to this!).

Reply:
My suggestion is not deletion here but rather retain the functionality from upstream https://github.com/electron/electron/pull/20397#discussion_r330804439 

```
//electron/chromium_src/BUILD.gn

static_library("chrome") {
...
sources = [
...
"//chrome/browser/ssl/security_state_tab_helper.h",
"//chrome/browser/ssl/tls_deprecation_config.h",
....
]
}
```

> My intuition is to try and leave as much Chromium source unchanged in the patch as possible

Yes that is the usual policy.

Reply:
I'm not sure it's that useful to list the header files in the sources list for our `chromium_src` target, GN will already pick up the .h dependency via `-MMD` analysis so listing the .h here is somewhat superfluous. Also it would be really hard to keep up with changes in #includes for every cc file we have here!

Reply:
Oops yeah that actually is a better option to follow. Thanks for noticing!